### PR TITLE
Add mention of "Enterprise Kapacitor", for HA alerting with InfluxEnterprise

### PR DESCRIPTION
**Pull request by mpursley**

### CODE REVIEW

File: content/docs/introduction/comparison.md [hunk 1]
```diff
@@ -132,14 +132,15 @@ boundaries like products, services, datacenters, or similar aspects.
 Independent servers (which can be run redundantly in parallel) may also give
 you better reliability and failure isolation.
 
-Kapacitor currently has no [built-in distributed/redundant
+Kapacitor's open-source release currently has no [built-in distributed/redundant
```
Comment (brian-brazil) [inline content/docs/introduction/comparison.md]: I'd like to keep the same style as above, where the open source options are compared first and then the commercial option after all that.

File: content/docs/introduction/comparison.md [hunk 2]
```diff
@@ -132,14 +132,15 @@ boundaries like products, services, datacenters, or similar aspects.
...
-Alertmanager's [High
-Availability](https://github.com/prometheus/alertmanager#high-availability)
-mode. In addition, Kapacitor can be scaled via manual sharding by the user,
-similar to Prometheus itself.
+alerting, or notifications.  [Enterprise Kapacitor](https://docs.influxdata.com/enterprise_kapacitor) 
+is Influx's commercial option for running a cluster of Kapacitors, for HA with
+alert deduplication.  Prometheus and the Alertmanager by contrast offer a 
+redundant option via running redundant replicas of Prometheus and using the 
+Alertmanager's [High Availability](https://github.com/prometheus/alertmanager#high-availability)
+mode. In addition, the open-source Kapacitor can be scaled via manual sharding
```
Comment (brian-brazil) [inline content/docs/introduction/comparison.md]: This is also the case for the commercial Kapacitor AFAICT. It is equivalent in this sense to Prometheus+AM.

File: content/docs/introduction/comparison.md [hunk 3]
```diff
@@ -132,14 +132,13 @@ boundaries like products, services, datacenters, or similar aspects.
 Independent servers (which can be run redundantly in parallel) may also give
 you better reliability and failure isolation.
 
-Kapacitor currently has no [built-in distributed/redundant
+Kapacitor's open-source release currently has no [built-in distributed/redundant
```
Comment (brian-brazil) [inline content/docs/introduction/comparison.md]: We can remove the "currently" and remove the link, as it looks like there won't be one.

File: content/docs/introduction/comparison.md [hunk 4]
```diff
@@ -132,14 +132,13 @@ boundaries like products, services, datacenters, or similar aspects.
...
-redundant option via running redundant replicas of Prometheus and using the
-Alertmanager's [High
-Availability](https://github.com/prometheus/alertmanager#high-availability)
-mode. In addition, Kapacitor can be scaled via manual sharding by the user,
-similar to Prometheus itself.
+alerting, or notifications.  Prometheus and the Alertmanager by contrast offer a 
+redundant option via running redundant replicas of Prometheus and using the 
+Alertmanager's [High Availability](https://github.com/prometheus/alertmanager#high-availability)
+mode. In addition, Kapacitor can be scaled via manual sharding by the user, similar 
+to Prometheus itself.
```
Comment (brian-brazil) [inline content/docs/introduction/comparison.md]: You've removed the mention of the commercial Kapacitor, which I thought was the point of this PR.

**Comment by mpursley** [comment]
@brian-brazil  Yeah, it didn't make sense to add the link to Enterprise Kapacitor at the bottom by it self.... 
So, I added links to Enterprise Influx and Enterprise Kapacitor at the bottom of the summary section.  I also removed the "currently" for open-source Kapacitor, and shortened up some extra long lines.

Please review again....

File: content/docs/introduction/comparison.md (hunk 4 above)
Comment (mpursley) [reply content/docs/introduction/comparison.md]: Well, the point of the PR was to clear up the "kapacitor doesn't support HA" line...  As it does now, with the commercial version at least.  

I just re-added the link for Enterprise Kapacitor and Enterprise Influx at the bottom of the summery section... 
Does this look better?

File: content/docs/introduction/comparison.md [hunk 5]
```diff
@@ -164,8 +163,12 @@ Where Prometheus is better:
 
 InfluxDB is maintained by a single commercial company following the open-core
 model, offering premium features like closed-source clustering, hosting and
-support. Prometheus is a [fully open source and independent project](/community/), maintained
-by a number of companies and individuals, some of whom also offer commercial
+support. For example, [Enterprise InfluxDB](https://docs.influxdata.com/enterprise_influxdb/v1.3/)
```
Comment (brian-brazil) [inline content/docs/introduction/comparison.md]: I think it's a little unfair to Influx to only mention the commercial Kapacitor down here, it makes it seem as though there's no commercial clustered option. I'd put this up in the Kapacitor section.

This section is more about how the projects are setup.

Comment (mpursley) [reply content/docs/introduction/comparison.md]: Ok, I've added lines for "Enterprise Kapacitor" in the top and bottom sections now... Please review again.

**Comment by mpursley** [comment]
@brian-brazil I've added lines for "Enterprise Kapacitor" in the top and bottom sections now... Please review again.

File: content/docs/introduction/comparison.md (hunk 5 above)
Comment (brian-brazil) [reply content/docs/introduction/comparison.md]: The changes to this paragraph should be removed.

File: content/docs/introduction/comparison.md [hunk 6]
```diff
@@ -132,14 +132,16 @@ boundaries like products, services, datacenters, or similar aspects.
...
-alerting, or notifications. Prometheus and the Alertmanager by contrast offer a
-redundant option via running redundant replicas of Prometheus and using the
-Alertmanager's [High
-Availability](https://github.com/prometheus/alertmanager#high-availability)
-mode. In addition, Kapacitor can be scaled via manual sharding by the user,
-similar to Prometheus itself.
+Kapacitor's open-source release has no built-in distributed/redundant options for 
+rules,  alerting, or notifications.  The open-source release of Kapacitor can 
+be scaled via manual sharding by the user, similar to Prometheus itself.
+Influx does also now offer [Enterprise Kapacitor](https://docs.influxdata.com/enterprise_kapacitor), which supports an
```
Comment (brian-brazil) [inline content/docs/introduction/comparison.md]: Hmm, you've an Irish influence on your grammar. "Influx offers" is sufficient.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**